<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>字幕翻译</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QComboBox" name="fromLanguageBox">
              <item>
               <property name="text">
                <string>自动检测</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>中文</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>英语</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>日语</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="textFromEdit">
              <property name="columnCount">
               <number>1</number>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderStretchLastSection">
               <bool>false</bool>
              </attribute>
              <column/>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QComboBox" name="translateAPIBox">
              <item>
               <property name="text">
                <string>百度翻译</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>谷歌翻译</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="translateButton">
              <property name="text">
               <string>翻译</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QComboBox" name="toLanguageBox">
              <item>
               <property name="text">
                <string>自动检测</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>中文</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>英语</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>日语</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="textToEdit">
              <property name="columnCount">
               <number>1</number>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderHighlightSections">
               <bool>true</bool>
              </attribute>
              <column/>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
       <zorder></zorder>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
   <zorder>stackedWidget</zorder>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openFile"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>翻译</string>
    </property>
    <addaction name="saveSubtitle"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openFile">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="saveSubtitle">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
